src/vm/disassembler.rs

```diff
@@ -396,51 +412,52 @@ mod tests {
               fn init(greeting) {
                 this.greeting = greeting;
               }
-              
+
               fn greet(name) {
                 return this.greeting + \" \" + name;
               }
             }
-              
+
             let greeter = Greeter(\"Hello\");
-              
+
             print greeter.greet(\"World\"); // out: Hello World",
             concat!(
                 "0000 OP_CLASS            0 == 'Greeter'\n",
                 "0002 OP_DEFINE_GLOBAL    0 == 'Greeter'\n",
                 "0004 OP_GET_GLOBAL       0 == 'Greeter'\n",
                 "0006 OP_NIL\n",
-                "0007 OP_FIELD            1 == 'greeting'\n",
-                "0009 OP_POP\n",
-                "0010 OP_GET_GLOBAL       0 == 'Greeter'\n",
-                "0012 OP_CLOSURE          2 == '<fn init arity=1>'\n",
+                "0007 OP_FIELD public    1 == 'greeting'\n",
```

Comment:
This is a bug. The index operand should be aligned with the others if the access modifier is `public` or `private`.